### ISSUE TITLE
Add flowbite tailwind plugin

### ISSUE THREAD

**Issue by countradooku** [windicss/plugins#10]
Can we have a flowbite plugin FOR windi? I Ried the transform api but it doesn't work well

https://github.com/themesberg/flowbite

**Comment by AlessioGr** [comment]
Yup I'd need that too. Would love to use Flowbite with either Windi or UnoCSS

**Comment by AlessioGr** [comment]
I made a Windi plugin here: https://github.com/AlessioGr/flowbite-windicss-plugin

Not sure if it works for everything - but so far I didn't notice any issues
